pom.xml

```diff
@@ -0,0 +1,246 @@
+<?xml version="1.0" encoding="UTF-8"?>
```

Comment:
Since this is a new repo, let's enable checkstyle and findbugs for the repo during build time.

Reply:
fixed with https://github.com/wso2/identity-content-repository/pull/1/commits/12e2eeaecc725e061a93b71f195c99fe7d913b2f